object/check.go

```diff
@@ -395,6 +408,10 @@ func CheckUserPassword(organization string, username string, password string, la
 				return nil, fmt.Errorf(i18n.Translate(lang, "check:The user: %s doesn't exist in LDAP server"), username)
 			}
 
+			if err.Error() == "user is disabled" {
+				return nil, fmt.Errorf(i18n.Translate(lang, "check:The user is forbidden to sign in, please contact the administrator"))
```

Comment:
What do you think about user update in Casgate?
if we set a user as blocked, we can, for example, filter the list of users in UI

Reply:
I think we can do it, it won't be a problem
we will just need to add a new field to the user table in db

Reply:
> I think we can do it, it won't be a problem we will just need to add a new field to the user table in db

field already exist user.IsForbidden

Reply:
i agree, this field is suitable